packages/expo-ui/src/swift-ui/Picker/index.tsx

```diff
@@ -47,14 +53,33 @@ function transformPickerProps(props: PickerProps): NativePickerProps {
     modifiers,
     ...(modifiers ? createViewModifierEventListener(modifiers) : undefined),
     ...restProps,
-    variant: props.variant ?? 'segmented',
-    color: props.color,
   };
 }
 
 /**
- * Displays a native picker component. Depending on the variant it can be a segmented button, an inline picker, a list of choices or a radio button.
+ * Displays a native picker component
+ * @example
+ * ```tsx
+ * <Picker modifiers={[pickerStyle('segmented')]}>
+ *   <Text modifiers={[tag('option1')]}>Option 1</Text>
+ *   <Text modifiers={[tag(0)]}>Option 3</Text>
+ * </Picker>
+ * ```
  */
 export function Picker(props: PickerProps) {
-  return <PickerNativeView {...transformPickerProps(props)} />;
+  const { label, children, ...restProps } = transformPickerProps(props);
+  if (typeof label === 'string') {
```

Comment:
since we introduce breaking change already, not sure if that's meaningful to keep the label string support?
i feel that to remove label support to make the native code cleaner a little bit